backend/runner/runner.go

```diff
@@ -39,6 +42,13 @@ import (
 	"github.com/TBD54566975/ftl/internal/unstoppable"
 )
 
+const name = "ftl.xyz/ftl/go/runner"
+
+var (
+	meter           = otel.Meter(name)
+	instanceCounter metric.Int64UpDownCounter
+)
```

Comment:
Could we pull this out into a separate file like you did for the controller? It would be best for all the otel stuff to live in dedicated files. That makes it more obvious to a reader what the pattern is, so when we add new features in the future, it's easy to copy the instrumentation pattern